### PR TITLE
ARTESCA-6420: add releases workflow

### PR DESCRIPTION
**Pull request by ghost**
Adding the release workflow for arsenal-networking.

First release output: https://github.com/scality/arsenal-networking/releases/tag/v0.0.1-alpha.1
Release run: https://github.com/scality/arsenal-networking/actions/runs/3539756617

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    needs:
+    - verify-release
+    steps:
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: Release ${{ github.event.inputs.tag }}
+          tag_name: v${{ github.event.inputs.tag }}
```
Comment (welansari) [inline .github/workflows/release.yaml]: I think it's maybe better to stick with how we name tags in other repos to keep things more consistent (i.e without the `v`) ?

Comment (ghost) [reply .github/workflows/release.yaml]: Good catch and I agree. Updated in: https://github.com/scality/arsenal-networking/pull/3/commits/09083c281df324da03d43ffbe46fb178cbda4feb

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        with:
+          ref: ${{ github.ref }}
+      
+      - name: Fetch tags
+        run: git fetch --tags
+      
+      - name: 'Check if tag was already created'
+        shell: bash
+        run: >
+          git show-ref --tags v${{ github.event.inputs.tag }} --quiet \
```
Comment (welansari) [inline .github/workflows/release.yaml]: here as well 🙂

Comment (ghost) [reply .github/workflows/release.yaml]: https://github.com/scality/arsenal-networking/pull/3/commits/26c9e562b203de94134540b5c3173f0caab0f9e2 :smiling_face_with_tear:

**Review by welansari — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
I added the nodejs engine version in package.json and a documentation file on how to release the project.

**Pull request merged**